extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.css

```diff
@@ -13,3 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-img-quiz-answer-choice {
+  position: absolute !important;
+  height: 1.5em !important;
+  width: 1.5em !important;
+  bottom: 0.5em !important;
+  right: 0.5em !important;
```

Comment:
Does this work for RTL (right to left) templates and languages?

Reply:
There's no RTL support yet. I was planning on including all RTL-related code in a future pull request, which includes changing this code for the position of the letters and changing the order of the options to look like this: [Who Wants to be a Millionaire Arabic](https://www.google.com/search?q=who+wants+to+be+a+millionaire+arabic&tbm=isch&ved=2ahUKEwiu1cuowL3xAhVEtVMKHZ04ALgQ2-cCegQIABAA&oq=who+wants+to+&gs_lcp=CgNpbWcQARgBMgUIABCxAzIECAAQQzICCAAyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAOggIABCxAxCDAToHCAAQsQMQQ1CKvgJYpMsCYILVAmgAcAB4AIABlgGIAZMOkgEEMC4xM5gBAKABAaoBC2d3cy13aXotaW1nwAEB&sclient=img&ei=c2rbYO7PJsTqzgKd8YDACw&bih=821&biw=1410&rlz=1CAOLRV_enUS956US956#imgrc=PM3NztWPd6NSLM)

Reply:
Is this tracked anywhere?

Reply:
Matias just made this issue https://github.com/ampproject/amphtml/issues/35110 and I should have a PR ready for it basically right after this PR is merged